.github/workflows/installdaprwin.yml

```diff
@@ -20,9 +20,21 @@ on:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
+jobs:
+  pre_job:
+    name: Skip Duplicate Actions
+    runs-on: ubuntu-latest
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@v3.4.0
+        with:
+          paths_ignore: '["**.md", ".codecov.yaml", ".github/**/**.yml"]'
```

Comment:
Same as above.

Reply:
Not all, some like stale bot would work before merge and tests for those changes make no sense. 

We can rename tests yaml file to achieve it.